packages/react-instantsearch/src/widgets/Panel.js

```diff
@@ -0,0 +1,37 @@
+import Panel from '../components/Panel.js';
+
+/**
+ * The Panel component will add a title to a widget. It will also handle when a wrapped widget doesn't have any refinement:
```

Comment:
No but users might use it like that:

```jsx
<Panel>
  <YourCart />
</Panel>
```

```jsx
<Panel>
  <Partners>
     <UPS/>
  </Partners>
</Panel>
```

If there's no constraint on Panel usage (= can be used on any widget) then we do not need to list the widgets that have a specific behavior when used.

Otherwise as a user you might have to parse through the whole list to know if you can use Panel (while you can definitely use it anywhere). And we will forget about maintaining the list.